minigalaxy/paths.py

```diff
@@ -11,6 +11,7 @@
 CACHE_DIR = os.path.join(os.getenv('XDG_CACHE_HOME', os.path.expanduser('~/.cache')), "minigalaxy")
 
 THUMBNAIL_DIR = os.path.join(CACHE_DIR, "thumbnails")
+DESKTOP_DIR = os.path.expanduser("~/.local/share/applications")
```

Comment:
This doesn't look like desktop dir. It is rather application menu directory. One cleaver way I know to obtain desktop dir (for every language) is as follow:
"
import subprocess
subprocess.check_output(['xdg-user-dir', "DESKTOP"]).decode("utf-8").strip()
"
Nevertheless adding menu application launcher might be a good idea too.

Reply:
I would recommend using ``DESKTOP_DIR = os.path.join(os.getenv('XDG_DATA_HOME', os.path.expanduser('~/.local/share')) + /applications`` here, so we adhere to the XDG standard.